src/_assets/css/main.scss

```diff
@@ -1074,18 +1068,11 @@ body.hide_toc {
       width: calc(100vw - 300px);
     }
   }
-  #toc {
+  #site-toc--side {
```

Comment:
This looks like you're nesting two `#site-toc--side` rules, one in another. Is that the idea?

Reply:
The `#toc` id is replaced by the `#site-toc--side` id, there is no nesting here. There are `li.toc-entry` entries later in the rules, but those are for the internal structure of the table of contents.